bot/src/main/java/org/opendaylight/bot/jenkins/Client.java

```diff
@@ -0,0 +1,16 @@
+/*
+Initial Jenkins Client API 
+*/
+
+package org.opendaylight.bot.jenkins;
+
+import com.cdancy.jenkins.rest.JenkinsClient;
+
+public class Client {
+
+  JenkinsClient client = JenkinsClient.builder()
+  .endPoint("http://127.0.0.1:8080")
+  .credentials("admin:password") 
```

Comment:
what should be in place of password?
An encrypted version of it in string format?

Reply:
add to bot/src/main/java/org/opendaylight/bot/BotConfiguration.java and pass it through to here

forget about having to encrypt it - not neccessary in this context